lib/browser/api/menu.js

```diff
@@ -189,6 +189,19 @@ Menu.prototype.closePopup = function (window) {
   }
 }
 
+Menu.prototype.getMenuItemById = function (id) {
+  let items = this
+  if (items instanceof Menu) items = this.items
```

Comment:
I might be missing something, but why not go with `this.items` directly?

Reply:
oops i forget to update that from when i refactored it to remove the `items` parameter...fixing now